cypress/features/regression/confirm.feature

```diff
@@ -95,3 +86,34 @@ Feature: Confirm component
       | error     | iconError    |
       | warning   | iconWarning  |
       | empty     | iconEmpty    |
+
+  @positive
+  Scenario: cancel button type is set to tertiary
+    Given I open "Confirm" component page "cancel button type" in no iframe
+    When I open component preview in noIFrame
+    Then cancel button type is set to "tertiary"
+
+  @positive
+  Scenario: buttons have destructive css properties
+    Given I open "Confirm" component page "destructive" in no iframe
+    When I open component preview in noIFrame
+    Then cancel button type is set to "destructive"
+      And confirm button type is set to "destructive"
+  
+  @positive
+  Scenario: cancel button is disabled
```

Comment:
```suggestion
  Scenario: Cancel button is disabled
```

Reply:
it is not a component but only button name

Reply:
updated